src/bandwidthRtc.ts

```diff
@@ -270,7 +271,7 @@ class BandwidthRtc {
     }
   }
 
-  private setupNewPeerConnection(peerConnection: RTCPeerConnection, endpointId: string, mediaTypes: MediaType[], alias?: string): void {
+  private setupNewPeerConnection(peerConnection: RTCPeerConnection, endpointId: string, mediaTypes: MediaType[], alias?: string, participantId?: string): void {
```

Comment:
If we added the participant ID to the response from requestToPublish (https://github.com/Bandwidth/webrtc-browser/blob/master/src/bandwidthRtc.ts#L89) we'd be able to add participantId to the RtcStream that's returned from publish(). 

The backend change would be pretty simple from what I can tell. Thoughts on this?

Reply:
On the surface I'm not opposed, but I think it'd be good to take a step back and think this through at a slightly higher level. The reason I say that is getting back my own participant id from a publish request feels a little weird, and makes me wonder if we should be setting that earlier or at a higher level.

Technically we could crack the JWT open and get the participant id ourselves, but I'm thinking the most appropriate way to do this might be to have `connect` return the `participantId` and then set that as a private variable in `bandwidthRtc` so it's globally available; something like this:

```
class BandwidthRtc {
  private participantId?: string;

  async connect(...): Promise<string> {
    ...
    participantId = await this.signaling.connect(...);
    return participantId;
  }

  getParticipantId(): string {
    // TODO: behavior if unset?
    return participantId;
  }
  ...
}
```

That's my initial thought at least, I haven't spent a ton of time thinking it through. On the other hand, as a user, do I want to know that a stream has my `participantId` (presumably meaning I have to keep tabs on what my id is somewhere in my application) or do I really just want one or more flags that I can check to see if a stream is a publication or subscription? I'm really not sure... the `participantId` approach is probably more flexible, but not necessarily more usable. Depends on use cases I guess.

All that being said as I've talked this through I do see value in making `participantId` available as a member of `BandwidthRtc` in general (with a public getter as well, could see that being handy), so might as well go ahead and slap that on published streams, and no reason _not_ to return it from `requestToPublish`, although I think it'd be good to already have it available from the initial `connect` call.

This turned into a bit of a novel/stream of consciousness, happy to talk about it if it doesn't make sense.